src/StratisMasternodeDashboard/wwwroot/js/nodeevent.js

```diff
@@ -125,5 +128,29 @@ function ConnectAndReceiveSignalRServerHub(signalRPort) {
             }
         }
 
+        var sidechainconnections = '';
+        if (message.nodeEventType.includes("Stratis.Bitcoin.EventBus.CoreEvents.PeerConnectionInfoEvent")) {
+            var inbountCount = 0;
+            var type;
+            var filtered = message.peerConnectionModels.filter(function (d) {
+                if (d.inbound) {
+                    inbountCount++;
+                }
+            });
+            document.getElementById('lblSidechainTotalConnectedNode').innerHTML = ` ${message.peerConnectionModels.length}` + ' /';
+            document.getElementById('lblSidechainTotalInNode').innerHTML = inbountCount + ' /';
+            document.getElementById('lblSidechainTotalOutNode').innerHTML = (` ${message.peerConnectionModels.length}` - inbountCount);
+
+            var peerconnections = message.peerConnectionModels;
+            for (var i = 0; i < peerconnections.length; i++) {
+                sidechainconnections += "<tr>";
+                sidechainconnections += "<td class='text-left' style='width: 250px;'>" + peerconnections[i].address + "</td>";
+                sidechainconnections += "<td class='text-center' style='width: 150px;'>" + peerconnections[i].inbound + "</td>";
```

Comment:
I believe this was dispaying `outbound` or `inbound` before. It should still display it.

Reply:
Currently in signalR event, we have parameter name inbound with value true or false. Maybe we can change Parameter name Type and value inbound or outbound based on true/false while publishing the signalR event in the Fullnode. I will check whether we can do this in dashboard or Fullnode.

Reply:
It's done from dashboard. Thanks!

Reply:
Yes I checked the full node code and I saw that that value is a `bool`. In this case please put an `if else` here... if inbound is true then display `inbound` else `outbound`.